surfer/viz.py

```diff
@@ -294,17 +294,26 @@ class Brain(object):
         in different viewing panes.
     surf :  geometry name
         freesurfer surface mesh name (ie 'white', 'inflated', etc.)
-    curv : boolean
-        if true, loads curv file and displays binary curvature
-        (default: True)
     title : str
         title for the window
-    cortex : str or tuple
-        specifies how binarized curvature values are rendered.
-        either the name of a preset PySurfer cortex colorscheme (one of
-        'classic', 'bone', 'low_contrast', or 'high_contrast'), or the
-        name of mayavi colormap, or a tuple with values (colormap, min,
-        max, reverse) to fully specify the curvature colors.
+    cortex : str, tuple, dict, or None
+        Specifies how the cortical surface is rendered. Accepts
+        specification of a colormap (in which case binarized curvature
+        values are rendered) or a color (in which case the cortical
+        surface is rendered without binarized curvature values).  Can
+        be set to: (1) the name of one of the preset cortex styles
+        ('classic' [default], 'high_contrast', 'low_contrast', or
+        'bone'), (2) the name of a colormap, (3) a tuple with four
+        entries for (colormap, vmin, vmax, reverse) indicating the
+        name of the colormap, the min and max values respectively and
+        whether or not the colormap should be reversed, (4) a valid
+        color specification (such as a 3-tuple with RGB values or a
+        valid color name), or (5) a dictionary of keyword arguments
+        that is passed on to the call to surface. If set to `None`,
+        color is set to (0.5, 0.5, 0.5) rendering a gray brain without
+        binarized curvature.
```

Comment:
Can you make this prettier (and easier to read) with proper ReStructuredText formatting like:
```
cortex : ...
    Specifies how the cortical surface is rendered. Options:

        1. A color-like argument to render the cortex as a single
           color, e.g. ``'red'`` or ``(0.1, 0.4, 1.)``.
        2. A colormap, in which binarized curvature ...
        3. ...
        ...
```
Why even allow `None`, isn't it the same as passing `(0.5, 0.5, 0.5)`?